spec/rubocop/cop/rspec/capybara/visibility_matcher_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::VisibilityMatcher do
+  subject(:cop) { described_class.new(config) }
```

Comment:
`subject(:cop) { described_class.new }` is enough, since the cop is not configurable.